App.js

```diff
@@ -89,11 +89,19 @@ const App = () => {
     } catch (err) { console.log(err) }
   }
 
-  // Get Alert Level for Current Council State from GQL
+  // get the alert level for the selected council, using the data in councilList
   async function fetchAlertLevel() {
     try {
-      const alert = await API.graphql(graphqlOperation(getCouncil, {id: council}));
-      setAlertLevel(alert.data.getCouncil.level);
+      let level = 0;
+
+      for (let area in councilList){
+        if (councilList[area].value === council){
+          level = councilList[area].level;
+          break;
```

Comment:
Looks good - just thinking though, tightly couples it to the "council" state should we pass the council we're looking for instead?